ee/ucp/authorization/create-teams-with-ldap.md

```diff
@@ -36,23 +33,26 @@ before the members of the team are fully synced.
 ### Match Group Members (Direct Bind)
 
 This option specifies that team members should be synced directly with members
-of a group in your organization's LDAP directory. The team's membership will by
+of a group in your organization's LDAP directory. The team's membership will be
```

Comment:
Suggest editing the first sentence to read: "The Direct Bind option specifies that team members be synced directly with member of a group in the organization's LDAP directory."